scripts/dev/e2e.py

```diff
@@ -123,11 +123,11 @@ def build_and_push_e2e(repo_url: str, tag: str, path: str) -> None:
     build_and_push_image(repo_url, tag, path, "e2e")
 
 
-def build_and_push_prehook(repo_url: str, tag: str, path: str) -> None:
+def build_and_push_version_upgrade_hook(repo_url: str, tag: str, path: str) -> None:
     """
-    build_and_push_prehook builds and pushes the pre-stop-hook image.
+    build_and_push_version_upgrade_hook builds and pushes the pod-deleter image.
```

Comment:
we're no longer calling it the `pod-deleter` image right?

Reply:
We sure aren't, thank you!